apps/www/components/Profile/Portrait.js

```diff
@@ -77,27 +76,30 @@ const PortraitSelector = withT(
     isMe,
     values,
     errors,
-    onChange
+    onChange,
   }) => {
     useEffect(() => {
       if (isMandadory && !values.portrait && !user.portrait) {
         onChange({
           errors: {
-            portrait: t('profile/portrait/empty')
-          }
+            portrait: t('profile/portrait/empty'),
+          },
         })
       }
     }, [isMandadory])
 
+    const [colorScheme] = useColorContext()
+
     const preview = isEditing && values.portraitPreview
     const imgUrl =
       values.portrait !== undefined ? values.portraitPreview : user.portrait
     const img = (
       <span
         {...styles.img}
+        {...colorScheme.set('backgroundColor', 'hover')}
```

Comment:
Most of the PR ist just linter. This is the relevant change, including removal of background-color on line 12